### PR TITLE
fix: add passIfNoBaseline to snyk-delta result

### PR DESCRIPTION
**Pull request by mathild3r**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [ ] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

_Explain why this PR exists_

Add the passIfNoBaseline to the result for snyk-prevent-commit-status. 

### Notes for the reviewer

_Instructions on how to run this locally, background context, what to review, questions…_

### More information

- [Jira ticket SC-0000](https://snyksec.atlassian.net/browse/SC-0000)
- [Link to documentation](https://github.com/snyk/snyk-delta/wiki/)

### Screenshots

_Visuals that may help the reviewer_

### CODE REVIEW

File: src/lib/index.ts [hunk 1]
```diff
@@ -101,6 +102,8 @@ const getDelta = async(snykTestOutput: string = '', debugMode = false):Promise<S
       
       newVulns = typedSnykTestJsonResults.vulnerabilities.filter(x => x.type != "license")
       newLicenseIssues = typedSnykTestJsonResults.vulnerabilities.filter(x => x.type == "license")
+
+      passIfNoBaseline = true
```
Comment (aarlaud) [inline src/lib/index.ts:106]: That should be an option set by the user. Some folks will want to have the default behavior while some others will want to pass if no baseline

File: src/lib/snyk/issues.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ const getNewIssues = (
   const debug = getDebugModule();
 
   const MonitoredIssues = snykProject;
-  debug(`Monitored snapshot had %d issues`, MonitoredIssues.length);
+  //(`Monitored snapshot had %d issues`, MonitoredIssues.length);
```
Comment (aarlaud) [inline src/lib/snyk/issues.ts:24]: why are you dropping this debug?

**Review by aarlaud — APPROVED** [review]

File: src/lib/snyk/issues.ts (hunk 2 above)
Comment (mathild3r) [reply src/lib/snyk/issues.ts:24]: ah that was just because I wanted less debug lines. changing that now.

**Pull request merged**